frontend/src/pages/destructionlist/abstract/BaseListView.tsx

```diff
@@ -176,7 +177,21 @@ export function BaseListView<T extends Zaak = Zaak>({
           },
         ] as ButtonProps[])
       : [];
-    return [...dynamicItems, ...fixedItems];
+    const clearFiltersItem = {
+      children: (
+        <>
+          <Solid.XCircleIcon />
+          Filters wissen
+        </>
+      ),
+      variant: "warning",
+      wrap: false,
+      onClick: () => {
+        const pageParam = searchParams.get("page") || "1";
+        setSearchParams({ page: pageParam });
```

Comment:
This will also clear someOtherUnrelatedParam=someOtherValue (for example) which shouldn't be the case.